validator/validator-main.protoascii

```diff
@@ -1887,6 +1887,7 @@ tags: {
   attrs: { name: "border" }
   attrs: { name: "height" }
   attrs: { name: "width" }
+  attrs: { name: "sizes" }
```

Comment:
`sizes` is already a universal attribute in AMP and given we don't need `"native-sizes"` per comment above, there shouldn't be a need for validation changes.